src/test/integration.js

```diff
@@ -21,13 +25,15 @@ describe('Integration', () => {
     expect(res.body).toHaveProperty('task.id')
     taskID = res.body.task.id
     expect(res.body).toHaveProperty('task.timestamp')
-    expect(res.body).toHaveProperty('task.progress', 0)
-    expect(res.body).toHaveProperty('task.completed', false)
+    expect(res.body).toHaveProperty('task.progress') // may be 0 or 1
```

Comment:
Is there a timer we can control here to reliably have these just being the initial state of 0/false?

Reply:
I'm also more surprised that it's completing immediately than wanting to add a delay, but fwiw Jest supports mocking control of timers https://jestjs.io/docs/en/timer-mocks to make that reliable in tests. So if adding a (mocked) delay to verify initial state vs completed state reliably, that's how I'd go, possibly breaking this out to a new test to clarify the intention of testing initial state.